features/frameworks/TARGET_PSA/pal/pal_crypto_config.h

```diff
@@ -408,4 +404,4 @@
 
 #include "pal_crypto_config_check.h"
 
-#endif /* _PAL_CRYPTO_CONFIG_H_ */
```

Comment:
what is the reason for this change?

Reply:
Bug in the code, user config should augment the mbedtls one not replace it